vllm/worker/model_runner.py

```diff
@@ -921,7 +961,10 @@ def load_model(self) -> None:
                                    parallel_config=self.parallel_config,
                                    scheduler_config=self.scheduler_config,
                                    cache_config=self.cache_config)
-
+        model_forward_params = inspect.signature(self.model.forward).parameters
+        if ("inputs_embeds" in model_forward_params
+                and "inputs_embeds_masks" in model_forward_params):
+            self.model_supports_input_embeds = True
```

Comment:
Can we factor this out into a separate function (maybe inside `interfaces.py` for models)? Since this logic may be used for other model runners as well.